### PR TITLE
Add blob-generator

### PR DESCRIPTION
**Pull request by rejas**
# Thank you for opening a pull request to tiny-helpers.dev ❤️

Before you submit your PR, please take a look at the [contributing guidelines](https://github.com/stefanjudis/tiny-helpers/blob/master/CONTRIBUTING.md). :)

To make it easier for us to review and merge your PR, please make sure …

- [x] you only add one (!) new helper per pull request.
- [x] you have checked if an open PR already exists.
- [x] the submitted website is focused on a single, development related issue.
- [x] the `desc` field includes an "actionable sentence" (e.g. "Create something great" or "Transform something into something else").

Thank you! 🙇‍♂️

### CODE REVIEW

**Comment by Pandimadhubabu** [comment]
i want deploy to gh-pages

File: helpers/blob-generator.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Blob generator",
+  "desc": "Generate beautiful blob shapes for web and flutter apps. ",
+  "url": "https://blobs.app/",
+  "tags": [
+    "SVG"
+  ],
+  "maintainers": [
+    " lokesh-coder",
```
Comment (stefanjudis) [inline helpers/blob-generator.json:9]:
```suggestion
    "lokesh-coder",
```

File: helpers/blob-generator.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Blob generator",
+  "desc": "Generate beautiful blob shapes for web and flutter apps. ",
+  "url": "https://blobs.app/",
+  "tags": [
+    "SVG"
+  ],
+  "maintainers": [
+    "lokesh-coder",
+    " udaypydi",
```
Comment (stefanjudis) [inline helpers/blob-generator.json:10]:
```suggestion
    "udaypydi",
```

File: helpers/blob-generator.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  "name": "Blob generator",
+  "desc": "Generate beautiful blob shapes for web and flutter apps. ",
+  "url": "https://blobs.app/",
+  "tags": [
+    "SVG"
+  ],
+  "maintainers": [
+    "lokesh-coder",
+    "udaypydi",
+    " mvhk"
```
Comment (stefanjudis) [inline helpers/blob-generator.json:11]:
```suggestion
    "mvhk"
```

**Comment by stefanjudis** [comment]
Thanks @rejas! :) 

@Pandimadhubabu I'm not sure I understand? Can you explain?

**Comment by Pandimadhubabu** [comment]
https://github.com/mnrlive/tiny-helpers/tree/primary/.github

**Comment by Pandimadhubabu** [comment]
https://github.com/mnrlive/tiny-helpers/runs/1100628548?check_suite_focus=true

**Comment by stefanjudis** [comment]
@Pandimadhubabu Understood. I don't have the time to help with that and there will soon be API endpoints so that GH-pages are no reasonable alternative. :) Good luck.

**Pull request merged**